_posts/2021-08-18-debezium-ui-availability.adoc

```diff
@@ -0,0 +1,115 @@
+
+=== Create Connector - Additional properties (optional)
+The Additional properties are optional and can be summarized as follows:
+
+* *Filter definition* - entry of *regular expressions* which define the filters for inclusion/exclusion of the items that will be included for CDC.  The included items are displayed as the filters are entered and applied.
+* *Data options* - *Snapshot* and *Mapping* properties (optional).  The defaults can be viewed and changed if desired.
+* *Runtime options* - *Engine* and *Heartbeat* properties (optional).  The defaults can be viewed and changed if desired.
+
+=== Create Connector - Review
+The *Review* step provides a summary of the configuration that will be used to create the connector.  If happy with the selections, click 'Finish' to create the connector.  If the properties need adjustment, navigate back to the earlier steps.
+
+== Design Considerations
+
+The UI is implemented as a https://quarkus.io/[Quarkus]-based web application.  The backend is configured with the URL(s) of one or more Kafka Connect clusters and provides a REST interface for the frontend.  The frontend user interface uses https://reactjs.org/[ReactJS] as the primary technology, utilizing https://www.patternfly.org/v4/[Patternfly] react components and design patterns.
+
+As with everything in Debezium, the UI is fully open source (Apache License Version 2.0).  You can find the https://github.com/debezium/debezium-ui/[UI Source Code] under the Debezium organization on Github.
+
+== Trying It Out Yourself
+
+=== Debezium UI Container Image
+
+The Debezium UI https://hub.docker.com/r/debezium/debezium-ui[container image] is available for running the UI.
+E.g. run the following comand to start the UI and connect it to an existing Kafka Connect instance via Docker
+(where KAFKA_CONNECT_URI supplies a comma-separated list of the available Kafka Connect URI(s)):
+
+----
+$ docker run -it --rm --name debezium-ui -p 8080:8080 -e KAFKA_CONNECT_URI=http://connect:8083 debezium/debezium-ui:1.7
+----
+
+The UI connects to Kafka Connect via REST, so you need to make sure that the latter is reachable,
+e.g. by running both components on the same Docker network.
+
+[NOTE]
+====
+Currently, the UI connects to un-authenticated Kafka Connect instances.
+Also, there's no authorization or authentication implemented in the UI itself yet.
+Until that is the case, you should secure the components e.g. with your own proxy for authorization, if needed.
+====
+
+=== Self-contained example
+
+We have also created a self-contained example https://github.com/debezium/debezium-examples/tree/master/ui-demo[UI demo], which is included under https://github.com/debezium/debezium-examples[debezium-examples] on Github.  The UI demo includes a Docker Compose file which brings up several sources with data as well as the UI. Please refer to the https://github.com/debezium/debezium-examples/tree/master/ui-demo[README file] for more details on running the Debezium UI demo.
+
```

Comment:
@mdrillin, can you add a sentence here:

To learn more about the Debezium UI, please refer to the [reference documentation](...), putting in the actual link.